src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewSpaceCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors /
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import UIKit
+
+class DynamicTableViewSpaceCell: UITableViewCell {
+	private lazy var heightConstraint: NSLayoutConstraint = self.contentView.heightAnchor.constraint(equalToConstant: 0)
+
+	var height: CGFloat {
+		set {
+			if newValue == UITableView.automaticDimension {
```

Comment:
I am always a bit careful then comparing non-discrete numerical types like `CGFloat`s. What do you think about checking if `newValue` is roughly equal to `UITableView.automaticDimension`?

Reply:
This should be fine, as `UITableView.automaticDimension` will either be used as constant or it will not. Therefore a direct comparison should not be of concern for this matter. This is how UIKit handles the property internally as well.